test/bin/ci_phase_test.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# This script runs on the CI cluster, from the metal-tests step.
+
+set -xeuo pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source "${SCRIPTDIR}/common.sh"
+
+cd "${TESTDIR}"
+
+for scenario in ./scenarios/*.sh; do
+    scenario_name="$(basename "${scenario}" .sh)"
+    logfile="${SCENARIO_INFO_DIR}/${scenario_name}/run.log"
```

Comment:
I think we need to override the SCENARIO_INFO_DIR to point to the location in the artifact dir where the scenario data has been downloaded in the previous step. We might need a new variable in common.sh to make overriding all of the other directory variables easier.

Reply:
Yes, was just creating this commit to check whats needed from openshift/release and work it out from that starting point.
Base path for everything will also change once the microshift-test-payload image disappears, so we need to make it configurable in an easy way. Thanks for the heads up!

Reply:
Is the artifact dir shared between pods in the CI cluster? Or should we use the shared directory and then copy the content into the artifact directory separately?